docs/quick_start.rst

```diff
@@ -5,31 +5,40 @@ Quick Start
 
 .. highlight:: none
 
+The following will introduce you to the most fundamental mechanics of simexpal.
+The quick start guide will explain simexpal installation, as well as the
+features: instance and input parameter management, launching experiments, monitor
```

Comment:
I think it should be "monitoring experiments".

Reply:
I actually removed all the -ing forms here.